README.md

```diff
@@ -51,6 +55,8 @@ add her to the list of guests in the castle.
 Write your code here:
 ```ruby
 # code here
+  town.delete("Belle")
+  town|guests:|("Belle")
```

Comment:
What are these pipes (`|`)?

To access guests, you want to use `[ ]` notation. So something like this: `town[:guests]`.

Once you've access the array, you need to either use `.push` (same functionality as Javascript) or Ruby's `<<` shovel method, which works just like `.push`.